internal/cmd/restore.go

```diff
@@ -190,6 +193,9 @@ func (config pgBackRestRestore) Run(ctx context.Context) error {
 		config.PostgresCluster, types.ApplyPatchType, patch,
 		config.Patch.PatchOptions(patchOptions))
 	if err != nil {
+		if strings.Contains(err.Error(), "conflict") {
```

Comment:
⛏️ Perhaps this function identifies the error? https://pkg.go.dev/k8s.io/apimachinery/pkg/api/errors#IsConflict

```suggestion
		if apierrors.IsConfict(err) {
```

Reply:
Hmm, yeah, makes sense to me:

https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/util/managedfields/internal/conflict.go#L42
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go#L249

Reply:
Nice! Thanks for the suggestion. Here's a [PR](https://github.com/CrunchyData/postgres-operator-client/pull/82).